tests/functional/TrackingCest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace vigetbasetests\functional;
+
+use FunctionalTester;
+
+class TrackingCest
```

Comment:
Should this be `TrackingTest`?

Reply:
NVM: https://codeception.com/docs/02-GettingStarted

Reply:
Yeah I _think_ that the class naming doesn't matter and you can use anything, but I see both Cest and Test used